### DISCUSSION TITLE
"HideControlCenterButton"=dword:  not changing

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#595] [General]
EP  22000.376.40.18
Toggling "Hide Control Center Button" under System tray in Properties GUI  ( and restarting File Explorer) does not change  registry "HideControlCenterButton" dword value.
So Hide Control Centre button is not toggled correctly.

**Comment by valinet** [comment]
Yup, indeed, a small mistake when I shuffled around the things in the file. Good thing is that we have pre-release versions ;) Thanks for spotting it out. Let me know if there is something else that's out of place after my edits. Best is to take a look on https://github.com/valinet/ExplorerPatcher/blob/master/ExplorerPatcher/settings.reg from the repo, it's a registry file, easy to look on.

I pushed the fix, a new pre-release will be generated soon.

Merry Christmas!

**Reply by chip33** [reply]
Hi, I did just look through settings.reg quite carefully. No obvious missing entries or errors that I could see at all.

**Reply by valinet** [reply]
Thank you
